micarray_driver:
  ros__parameters:
    recordings:
      sample_rate: 16000
      bit_depth: 16
      num_channels: 6
      target_channels: [1, 2, 3, 4]
      format: 2 # This is the value of an enum factor"SND_PCM_FORMAT_S16_LE". Look enum _snd_pcm_format in pcm.h
      device_name: "hw:CARD=ArrayUAC10"
      recordings_duration_ms: 300
      ros_timer_period_ms: 500
      sound_speed: 343.0
    MUSIC:
      min_frequency: 700.0
      max_frequency: 1100.0
      spectrum_threshold: 9.8
      maintain_likelihood: 0.80
      activity_threshold: 0.80
      resolution_degree: 1
      microphone_arrangement: "micpos.csv"
